behave_django/testcase.py

```diff
@@ -5,11 +5,23 @@
 
 
 class BehaviorDrivenTestCase(StaticLiveServerTestCase):
+
+    """Test case attached to the context during behave execution
+
+    This test case prevents the regular tests from running.
+    """
+
     def runTest(*args, **kwargs):
         pass
 
 
 class ExistingDatabaseTestCase(BehaviorDrivenTestCase):
+
+    """ Test case used for the --use-existing-database setup
```

Comment:
I use an autoformatter and this is what it does lol. I'm honestly not sure if this is correct. I think PEP257 says that there should be a space if it's a class, but no space if it's a method or function. I made it more consistent with the other class docstrings.